src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -627,6 +643,8 @@ class Meta:
             "titel",
             "beschrijving",
             "registratiedatum",
+            "vernietigingsdatum",
+            "status",
```

Comment:
Should we add validation for `status.statustype == ZaakTypeInformatieObjectType.statustype`? 
There is no validation for it in reference implementation and no mentioning in OAS

Reply:
Hmm, the description doesn't describe it as a requirement, merely as some book keeping information. Good observation though!